DESIGN.md

```diff
@@ -0,0 +1,98 @@
+# Bulk Data Extract (BDE) Processor design
+
+## User requirements
+
+-  The _user interface_ is a read-only PostgreSQL database in AWS RDS, with
+   Azure authentication.
+-  _Accessible_ to all
```

Comment:
Should we add here that it's accessible to Koordinate / LDS platform? Can add these links for context.

https://data.linz.govt.nz/sets/category/full-landonline-dataset/
https://data.linz.govt.nz/data/category/property-ownership-boundaries/

Reply:
LDS is already in that wiki page; do we need to add more details?

Reply:
Oh sorry. Missed that. Yes, that's fine.